scrapers/GirlsOutWest.yml

```diff
@@ -0,0 +1,44 @@
+name: GirlsOutWest
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://tour.girlsoutwest.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class='trailerTitle']/div/h3/text()
+        replace:
```

Comment:
since you used the new format for the Date field it should also be the same for the title and image field
eg 
```yaml
      Title:
        selector: //div[@class='trailerTitle']/div/h3/text()
        postProcess:
          - replace:
            - regex: .+-
              with:
```